docs/src/main/paradox/scala/http/routing-dsl/directives/basic-directives/withExecutionContext.md

```diff
@@ -10,7 +10,7 @@
 
 ## Description
 
-Allows running an inner route using an alternative `ExecutionContext` in place of the default one.
+Allows running an inner route using an alternative `ExecutionContextExecutor` in place of the default one.
```

Comment:
I didn't even know that. I find it weird, actually. I think, the API should be different for Java and Scala. I created https://github.com/akka/akka-http/issues/1632 to improve the APIs.